### ISSUE TITLE
`<div onclick={{action` cases

### ISSUE THREAD

**Issue by miguelcobain** [runspired/ember-hammertime#15]
Can the AST walker handle `<div onclick={{action` cases?

**Comment by eriktrom** [comment]
@miguelcobain - yes although b/c there are no event listeners, and its not a link or button, it doesn't make much sense until javascript loads (and replaces the dom anyway) 

However - does work if u have a use case (from your project, ember paper):

```hbs
{{#paper-item onClick=onClick class=(concat "menu-item" (if active ' active') (if expanded ' expanded'))}}
  {{yield}}
  {{#if canExpand}}
    {{paper-icon 'expand-more' class='expand-icon'}}
  {{/if}}
{{/paper-item}}
```

--> ast transformed html (from server)

```html
<div id="ember461" class="ember-view"><md-list-item id="ember462" style="touch-action: manipulation; -ms-touch-action: manipulation; cursor: pointer;" class="menu-item ember-view md-clickable"><div class="md-no-style md-list-item-inner">
  Components
    <md-icon id="ember472" style class="expand-icon ember-view paper-icon md-font material-icons md-default-theme">expand_more</md-icon>

</div>
```

**Comment by runspired** [comment]
Resolved this in 1.1.1

Have fun @miguelcobain :)

**Comment by miguelcobain** [comment]
<3
